src/Orchard.Web/Modules/Orchard.OpenId/Services/OpenIdAuthenticationService.cs

```diff
@@ -19,14 +19,16 @@ public class OpenIdAuthenticationService : IAuthenticationService {
         private readonly IHttpContextAccessor _httpContextAccessor;
         private readonly IMembershipValidationService _membershipValidationService;
         private readonly IEnumerable<IOpenIdProvider> _openIdProviders;
+        private readonly IEnumerable<IUserDataProvider> _userDataProviders;
+        private readonly ISecurityService _securityService;
```

Comment:
I had to make changes here because of the new services being injected in FormsAuthenticationService